### PR TITLE
Fix bogus CLB/DES and R6 arrow in VERTREV pages

### PR DESCRIPTION
**Pull request by gizmo71**
### Description of Changes
The VERTREV page correctly had "RETURN" as L6 with an arrow for PPOS/origin/destination, but in waypoint variant had "CLB" as L6 and "DES" as R6, and all four had the right hand arrow enabled.

All four now have the arrow at R6 removed and the L6 label set to "RETURN", which I believe is correct behaviour for all of them.

### Checklist:

* [x] My changes follow the Contributing Guidelines. 
* [ ] My changes implement realistic features. 
* [ ] Please have a main Developer test my changes before merging. 
* [x] My changes are ready for merging.

### CODE REVIEW

**Comment by legoboyvdlp** [comment]
Hi,
I'm following the flight crew operating manual here:

image

The documents are quite unclear; do you have any other source on this? I took it that any waypoint VERT REV page should have CLB / DES until such time as we have a correct vertical flightplan functionality. [1]

But certainly, the arrow on R6 is a bug and shouldn't show unless there is a temporary flightplan so that would be good to fix. 

Thanks for the pull request again!

[1] The vert rev page is present but not actually functional for now :)

**Comment by merspieler** [comment]
@gizmo71 we'd like you to join us on mattermost since you seem to be interested in helping.

Any way I can contact you directly?

**Comment by gizmo71** [comment]
My reading of the manual is that "predictions are not computer" implies that the vertical revision is being made before the FMGS has enough data to work out where the top of climb and descent are, so "CLB*" or "DES*" have to be manually selected to give it a hint; if you look a page or two earlier in the manual (I assume it's the same or very similar to [this](https://www.theairlinepilots.com/forumarchive/a320/a320-fms-pages.pdf)) the "VERT REV at cruise WPT" just has "RETURN" as does the variant with a predicted missed constraint.

I get the impression from other training material (e.g. [A320DP](http://www.a320dp.com/A320_DP/nav-flight-management/sys-14.8.14.html)) that this isn't common. Technically it's probably _always_ the case right now in this model, but only because it doesn't really model the vertical flightpath at all yet.

If we're going to leave the CLB* OR DES* labels, then we need to at least make sure that R6 also functions like 'RETURN' on L6 would.

**Comment by legoboyvdlp** [comment]
Ah; that's fair enough. It also shows when a waypoint is in cruise.

I think the best thing to do now is to show CLB / DES right now but as you said make R6 also work. 

Vertical flightplan will be coming at some stage reasonably soon.  :)

**Comment by gizmo71** [comment]
Sounds like a plan; leave it with me and I'll tweak the pull request tomorrow and leave some comments behind about what future tweaks will be needed.

**Comment by gizmo71** [comment]
Turns out there was a single character typo preventing RSK6 returning to the FPLNA page like LSK6.

I gave up trying to get asterisks instead of arrows next _a la_ `*DES` and `CLB*`, or the word `OR` in amber text between them. I still think this is an edge case and that `RETURN` is the common path. Anyway, I left behind a load of comments about what it should look like eventually.

File: Nasal/MCDU/VERTREV.nas [hunk 1]
```diff
@@ -61,10 +61,16 @@ var vertRev = {
...
+			me.L6 = [" CLB", nil, "amb"];
 			me.R2 = ["RTA ", nil, "wht"];
 			me.R3 = ["[      ] ", "ALT CSTR  ", "blu"];
-			me.arrowsMatrix = [[0, 0, 0, 1, 1, 1], [0, 1, 0, 0, 0, 0]];
+			me.R6 = ["DES ", nil, "amb"];
+			# When the system does vertical planning, L6 should be RETURN and R6 not used if the MCDU knows the waypoint is during climb or descent.
+			# The CLB or DES prompts should only be shown for a vertical revision in the cruise phase.
+			# For now we fake it and allow the user to press either, which both act like RETURN.
+			# When CLB/DES are shown, a small "OR" should be shown between them.
+			# The 'arrows' for CLB/DES should actually be asterisks.
```
Comment (legoboyvdlp) [inline Nasal/MCDU/VERTREV.nas:72]: Asterisks require changes to the .svg . I will progressively add these to the missing pages. I know other pages need changes too.

**Review by legoboyvdlp — APPROVED** [review]

**Pull request merged**